english/happy-robot/happy-robot.md

```diff
@@ -2,3 +2,7 @@ A happy robot helps people.
 
 It was a dark and stormy night.  A scream is heard.  A happy robot, let's name him "George," wakes up to the sound.
 "Someone needs my help!"
+
+He jumps up, but before he rushes out the door he...
+a) Grabs a [weapon](robot-help/robot-weapon.md), or
+b) Enjoys some [refreshments](robot-help/refreshments.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.